specification/devcenter/data-plane/Microsoft.DevCenter/stable/2023-04-01/devcenter.json

```diff
@@ -3701,30 +3675,13 @@
     },
     "StorageProfile": {
       "type": "object",
-      "description": "Storage settings for the Dev Box's disks",
+      "description": "Storage settings for the Dev Box's disks.",
       "properties": {
         "osDisk": {
           "$ref": "#/definitions/OsDisk",
           "description": "Settings for the operating system disk."
         }
       }
-    },
-    "User": {
```

Comment:
Same question as above

Reply:
Not a model in the sdk neither. I think this is being deleted because we create the `User` model in ths tsp just for route propose.